widgets/valitext/org.eclipse.nebula.widgets.valitext.example/build.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2019 Uenal Akkaya <uenal.akkaya@sap.com>, Michael Gutfleisch <michael.gutfleisch@sap.com>
```

Comment:
We also need explicit consent from Michale Gutfleisch. Michael needs to comment on the pull request and also must sign the Eclipse Contributor agreement. We probably also need consent from the SAP company.